index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="description" content="Adryan Corcione portfolio">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@200..700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <h1>Adryan Corcione</h1>
+    <section id="about">
+        <figure>
+            <img src="https://images.squarespace-cdn.com/content/v1/656a1fc5ed64a607d3ecf423/7d719ba3-0bde-4065-91d6-40195bd202dc/new+headshot%283%29.png" alt="headshot of Adryan Corcione" width="300" height="300">
+        </figure>
+        <h2>About</h2>
+        <p>I'm a journalist and content marketer learning how to code. I hope to incorporate my love of pop culture into programming projects.</p>
+    </section>
+    <section id="experience"></section>
+        <h2>Experience</h2>
+            <ul>Content Marketing Manager, <i>Rocware</i> (2024-)</ul>
+            <ul>Copywriter, <i>FOLX Health</i> (2022-2023)</ul>
+            <ul>SEO Account Coordinator, <i>NisonCo</i> (2021)</ul>
+            <ul>Journalist, <i>Self-employed</i> (2016-2020)</ul>
```

Comment:
When listing items, try to use a `<ul>` tag as a parent container and `<li>` tags for each list item. (In case you have an hyperlink put the anchor tag `<a>` inside the `<li>` tag